drivers/storage/portworx/portworx_test.go

```diff
@@ -6235,6 +6379,111 @@ func manifestSetup() {
 	manifest.SetInstance(&fakeManifest{})
 }
 
+func TestSetTLSSpecDefaults(t *testing.T) {
+	// all filenames supplied
+	// setup
+	CACertFileName := stringPtr("testCA.crt")
+	serverCertFileName := stringPtr("testServer.crt")
+	serverKeyFileName := stringPtr("testServer.key")
+	// test
+	cluster := testutil.CreatePodSpecWithTLS(CACertFileName, serverCertFileName, serverKeyFileName)
+	s, _ := json.MarshalIndent(cluster.Spec.Security, "", "\t")
+	t.Logf("Security spec under test = \n, %v", string(s))
+	setTLSSpecDefaults(cluster)
```

Comment:
Can we call `driver.SetDefaultsOnStorageCluster` instead of this private method?
I think it's better to avoid testing private methods (if possible). Private methods tend to get refactored internally which shouldn't require a change in UTs.